src/compiler/commandLineParser.ts

```diff
@@ -603,6 +603,12 @@ namespace ts {
             category: Diagnostics.Source_Map_Options,
             description: Diagnostics.Emit_the_source_alongside_the_sourcemaps_within_a_single_file_requires_inlineSourceMap_or_sourceMap_to_be_set
         },
+        {
+            name: "allowUmdGlobalAccess",
+            type: "boolean",
+            category: Diagnostics.Module_Resolution_Options,

```

Comment:
You would want to set `affectsModuleResolution: true`

Reply:
I don't think it does, unless I misunderstand what `affectsModuleResolution` does

Reply:
then it needs affectsSemanticDiagnostics

Reply:
Added 👍